doc/09-object-types.md

```diff
@@ -1082,6 +1082,7 @@ Configuration Attributes:
   start\_time               | Timestamp             | **Required.** The start time as UNIX timestamp.
   end\_time                 | Timestamp             | **Required.** The end time as UNIX timestamp.
   duration                  | Number                | **Optional.** The duration as number.
+  sequence\_id              | String                | **Optional.** The generated sequence string. If multiple downtimes were created from the same API action, this sequence number allows to delete them in one shot. Available since v2.11.
```

Comment:
Colleagues, are there requests for similar features?

Reply:
I don't know of any.